lib/chess_club_web/middleware/authentication.ex

```diff
@@ -0,0 +1,14 @@
+defmodule ChessClubWeb.Middleware.Authentication do
+  @behaviour Absinthe.Middleware
+
+  def call(resolution, _config) do
+    case resolution.context do
+      %{current_user: _} ->
+        resolution
+
+      _ ->
+        resolution
+        |> Absinthe.Resolution.put_result({:error, "unauthenticated"})
```

Comment:
```suggestion
        Absinthe.Resolution.put_result(resolution, {:error, "unauthenticated"})
```

Reply:
Let's configure credo to find all instances of this and change them at once. Love the idea.